Kernel/Devices/Storage/USB/USBStorageController.cpp

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023, Leon Albrecht <leon.a@serenityos.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Singleton.h>
+#include <Kernel/Devices/Storage/StorageManagement.h>
+#include <Kernel/Devices/Storage/USB/USBStorageController.h>
+
+namespace Kernel::USB {
+
+static Singleton<StorageController> s_the;
```

Comment:
Going forward, we shouldn't have this. Please put a FIXME that we need to refactor the storage subsystem to allow standalone devices to be attached.